File: image/android/app/src/main/java/com/project/image/network/RetrofitClient.java
package com.project.image.network;

//public class RetrofitClient {
//}

object RetrofitClient {
private val retrofit = Retrofit.Builder()
        .baseUrl("https://your-api-url.com/") // Ganti dengan URL kamu
        .addConverterFactory(GsonConverterFactory.create())
        .build()

val api: ApiService = retrofit.create(ApiService::class.java)
}

File: request_id/android/app/src/main/java/com/project/requestid/network/RetrofitClient.kt
package com.project.requestid.network

import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

//class RetrofitClient {
//}

object RetrofitClient {
    private val retrofit = Retrofit.Builder().baseUrl("http://10.0.2.2:8080/request-id").addConverterFactory(GsonConverterFactory.create()).build()
    val webService: WebService = retrofit.create(WebService::class.java)
}